machado/management/commands/remove_analysis.py

```diff
@@ -8,38 +8,70 @@
 
 from django.core.management.base import BaseCommand, CommandError
 from django.core.exceptions import ObjectDoesNotExist
-from machado.models import Analysis, Analysisfeature, Feature, Featureloc
+from machado.models import Analysisprop, Analysis, Analysisfeature
+from machado.models import Feature, Featureloc, Organism
+from machado.models import Quantification, Acquisition
 
 
 class Command(BaseCommand):
     """Remove analysis."""
 
-    help = 'Remove analysis (CASCADE)'
+    help = 'Remove analysis (CASCADE). Also remove related features that are'
+    'multispecies (features that were loaded from the analysis file only).'
 
     def add_arguments(self, parser):
         """Define the arguments."""
-        parser.add_argument("--name", help="analysis.sourcename",
-                            required=True, type=str)
+        parser.add_argument("--name",
+                            help="source filename (analysisprop.value)",
+                            required=True,
+                            type=str)
 
     def handle(self,
                name: str,
-               verbosity: int=1,
+               verbosity: int = 1,
                **options):
         """Execute the main function."""
+        if verbosity > 0:
+            self.stdout.write('Deleting {} and every child'
+                              'record (CASCADE)'.format(name))
+        # get multispecies organism
+        multiorganism, created = Organism.objects.get_or_create(
+                abbreviation='multispecies',
+                genus='multispecies',
+                species='multispecies',
+                common_name='multispecies')
```

Comment:
Let's not use get_or_create here. I think the tool to remove something should not create any records.
If multispecies is not in the database, the code should not try to delete the feature.